### ISSUE TITLE
BaltCoGo: Add Tooltips

### ISSUE THREAD

**Issue by danfox01** [baltimorecounty/baltimorecountymd.gov-assets#120]
Some fields would benefit from additional information to help users understand them. For example, letting users know that anything entered in the "Description" field could be publicly viewed, or explaining why entering a valid email address is important. 

Tooltips should apply globally to all request types. They won't need to change based on which issue is selected.

Still undecided whether tool tips need to be included at the field level (e.g., "Description, Phone Number, " etc.) or if they would be more effective at the panel level (e.g., "How Can We Help?" "Provide Contact Information"). If this would change development requirements, let me know.

**Comment by martypowell** [comment]
@danfox01 are we using tooltips anywhere else on the site? (i think maybe on the coplogic pages, but that is very old)

**Comment by danfox01** [comment]
@martypowell, you're right about coplogic. I don't think we're using tool tips anywhere else at this time.  

If you're thinking of tackling this, I'm envisioning something simple with the flexibility to be associated with an fa icon or just a link on the page.
